modules/kapacitor-server/main.tf

```diff
@@ -0,0 +1,161 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# CREATE AN AUTO SCALING GROUP (ASG) TO RUN KAPACITOR
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "aws_autoscaling_group" "autoscaling_group" {
+  name = "${var.server_name}"
+
+  launch_configuration = "${aws_launch_configuration.launch_configuration.name}"
+  vpc_zone_identifier  = ["${var.subnet_ids}"]
+
+  min_size             = "1"
+  max_size             = "1"
```

Comment:
Don't need quotes around numbers. Also, why just a single server? Can this not be run as a cluster with a load balancer?

Reply:
I had a conversation with the InfluxSata team. We decided that open source Kapacitor was the way to go. Clustering capabilities are exclusive to the Enterprise edition

Reply:
Ah, right. Might be worth adding a comment about that.